psi4/driver/procrouting/proc.py

```diff
@@ -1446,20 +1462,22 @@ def run_dcft(name, **kwargs):
         ref_wfn = scf_helper(name, **kwargs)
 
     if (core.get_global_option("DCFT_TYPE") == "DF"):
-        aux_basis = core.BasisSet.build(ref_wfn.molecule(), "DF_BASIS_DCFT",
-                                        core.get_global_option("DF_BASIS_DCFT"),
+        aux_basis = core.BasisSet.build(ref_wfn.molecule(), "DF_BASIS_DCFT", core.get_global_option("DF_BASIS_DCFT"),
                                         "RIFIT", core.get_global_option("BASIS"))
         ref_wfn.set_basisset("DF_BASIS_DCFT", aux_basis)
 
-        scf_aux_basis = core.BasisSet.build(ref_wfn.molecule(), "DF_BASIS_SCF",
-                                            core.get_option("SCF", "DF_BASIS_SCF"),
-                                            "JKFIT", core.get_global_option('BASIS'),
-                                            puream=ref_wfn.basisset().has_puream())
```

Comment:
I'm not sure if I'm doing something wrong here, but I didn't touch this part of the code - after running `yapf --style .style.yapf psi4/driver/procrouting/proc.py -i` it splits these function calls differently!